agr_literature_service/api/models/mod_model.py

```diff
@@ -44,4 +44,11 @@ class ModModel(Base, AuditedModel):
         nullable=False
     )
 
+    reference_mod_md5sum = relationship(
+        "ReferenceModMd5sumModel",
+        lazy="joined",
```

Comment:
remove lazy="joined" to avoid loading a long list of objects every time we query a mod object